llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -2032,6 +2032,8 @@ bool ModuleAddressSanitizer::shouldInstrumentGlobal(GlobalVariable *G) const {
   if (G->isThreadLocal()) return false;
   // For now, just ignore this Global if the alignment is large.
   if (G->getAlign() && *G->getAlign() > getMinRedzoneSizeForGlobal()) return false;
+  if (G->getName() == "NoopCoro.Frame.Const")
```

Comment:
Seems attaching attach metadata on the global is nicer solution? 
See line 2020

Reply:
Also there is 
```
if (F.isPresplitCoroutine())
    return false;
```

Reply:
Thanks for the suggestion. Updated the GV metadata. I'll leave the function metadata unchanged because we still want to instrument post-split coroutine functions, and `isPresplitCoroutine()` can reflect the state.